tests/fixtures/basic/output/styles-daff78e51cc77ea97e75a7fd9c5f6fae.css

```diff
@@ -1,11 +1,11 @@
 .sample-img {
   width: 50px;
   height: 50px;
-  background-image: url(images/sample-b7caf246e908cd00d8011f370e3dd992.png);
+  background-image: url(images/sample-1f6b78f1b4667adc7e397f7bf94041ab.png);
```

Comment:
I've simply updated the affected fixture output files to use the appropriate MD5 values.